charts/azuredisk-csi-driver/templates/azuredisk-csi-driver.yaml

```diff
@@ -0,0 +1,141 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-azuredisk
+  namespace: {{ .Release.Namespace }}
+{{ include "adcd.labels" . | indent 2 }}
+    app: csi-azuredisk
+spec:
+  selector:
+    matchLabels:
+      app: csi-azuredisk
+  template:
+    metadata:
+{{ include "adcd.labels" . | indent 6 }}
+        app: csi-azuredisk
+    spec:
+      serviceAccountName: csi-driver-registrar
+      hostNetwork: true
+      containers:
+        - name: liveness-probe
+          imagePullPolicy: Always
+          volumeMounts:
+          - mountPath: /csi
+            name: socket-dir
+          image: quay.io/k8scsi/livenessprobe:v1.0.2
+          args:
+          - --csi-address=/csi/csi.sock
+          - --connection-timeout=3s
+        - name: driver-registrar
+          image: quay.io/k8scsi/csi-node-driver-registrar:v1.0.2
+          args:
+            - --v=5
+            - --csi-address=/csi/csi.sock
+            - --kubelet-registration-path=/var/lib/kubelet/plugins/disk.csi.azure.com/csi.sock
+          lifecycle:
+            preStop:
+              exec:
+                command:
+                  [
+                    "/bin/sh",
+                    "-c",
+                    "rm -rf /registration/disk.csi.azure.com /registration/disk.csi.azure.com-reg.sock",
+                  ]
+          env:
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: spec.nodeName
+          imagePullPolicy: Always
+          volumeMounts:
+          - mountPath: /csi
+            name: socket-dir
+          - mountPath: /registration
+            name: registration-dir
+        - name: azuredisk
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          args:
+            - "--v=5"
+            - "--endpoint=$(CSI_ENDPOINT)"
+            - "--nodeid=$(KUBE_NODE_NAME)"
+          ports:
+            - containerPort: 9808
+              name: healthz
+              protocol: TCP
+          livenessProbe:
+            failureThreshold: 5
+            httpGet:
+              path: /healthz
+              port: healthz
+            initialDelaySeconds: 30
+            timeoutSeconds: 10
+            periodSeconds: 30
+          env:
+{{- if eq .Values.provider "azure" }}
```

Comment:
I think this driver is only for azure, so remove it?

Reply:
I will do it.